compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/MetadataEncoding.hs

```diff
@@ -82,6 +93,40 @@ mapFunDep f (a, b) = (map f a, map f b)
 mapFunDepM :: Monad m => (a -> m b) -> (GHC.FunDep a -> m (GHC.FunDep b))
 mapFunDepM f (a, b) = liftM2 (,) (mapM f a) (mapM f b)
 
+-------------
+-- MINIMAL --
```

Comment:
```suggestion
-- MINIMAL PRAGMAS --
```
Just to make it slightly more explicit what this is about.

Reply:
Will do, thanks!